pkg/syncer/metadatasyncer.go

```diff
@@ -2909,3 +2915,84 @@ func startStoragePolicyQuotaCRInformer(ctx context.Context, cfg *restclient.Conf
 	}()
 	return nil
 }
+
+// storagePolicyUsageCRSync patches StoragePolicyUsage CRs for k8s PVs in Bound state
+func storagePolicyUsageCRSync(ctx context.Context, metadataSyncer *metadataSyncInformer) {
+	log := logger.GetLogger(ctx)
+	log.Debugf("Starting CnsVolumeOperationRequest CR sync.")
+	// Get K8s PVs in State "Bound".
+	k8sPVsInBoundState, err := getBoundPVs(ctx, metadataSyncer)
+	if err != nil {
+		log.Errorf("Failed to get PVs from kubernetes. Err: %+v", err)
+		return
+	}
+	k8sVolumesToNamespaceMap := make(map[string][]*v1.PersistentVolume)
+	for _, pv := range k8sPVsInBoundState {
+		k8sVolumesToNamespaceMap[pv.Spec.ClaimRef.Namespace] =
+			append(k8sVolumesToNamespaceMap[pv.Spec.ClaimRef.Namespace], pv.DeepCopy())
+	}
+	// Prepare Config and NewClientForGroup for cnsOperatorClient
+	restConfig, err := config.GetConfig()
+	if err != nil {
+		log.Errorf("failed to get Kubernetes config. Err: %+v", err)
+		return
+	}
+	cnsOperatorClient, err := k8s.NewClientForGroup(ctx, restConfig, cnsoperatorv1alpha1.GroupName)
```

Comment:
Can we use `metadataSyncer.cnsOperatorClient` instead?

Reply:
`metadataSyncer.cnsOperatorClient` is only initialized in GuestCluster code blocks. Using a local cnsoperator client and leaving this LOC unchanged